build.sbt

```diff
@@ -15,7 +15,7 @@ lazy val root = project
     stOutputPackage                         := "lucuma.bc",
     /* javascript / typescript deps */
     Compile / npmDependencies ++= Seq(
-      "broadcast-channel" -> "4.10.0"
+      "broadcast-channel" -> "5.3.0"
```

Comment:
And also broadcast-channel. Can we figure out which of these updates broke the imports?

Reply:
In the morning I'll try with the old ScalablyTyped.

Reply:
It turned out to be harder to downgrade the ScalablyTyped version because of dependencies on scala and sbt versions. But I returned broadcast-channel to 4.10.0 and still get the bad import in explore. I'll see if I can narrow down the ST version where the problem emerges.